### PR TITLE
Removed some unused properties in the manifest.json files

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Like it says on the tin, this PR cleans up some unused properties in the `manifest.json` files.

To do this, I used the following command:
```
find src/applications/ -name manifest.json -exec bash -c "jq 'del(.template) | del(.noRouting) | del(.hideInvitation) | del(.production)' {} > {}.new.json && mv {}.new.json {}" \;
```

### Breakdown
Find all the manifest files:
```
find src/applications -name manifest.json
```
Run a bash script on each file:
```
-exec bash -c "..."
```
Needed to `-exec bash -c` because I wanted to write to each individual file, and if I told `find` to `-exec` `jq` directly, and _then_ redirected the output to `{}`, it interpreted that as redirecting the output of all calls to `jq` to a file called `{}`.

Remove the unused properties:
```
jq 'del(.template) | del(.noRouting) | del(.hideInvitation) | del(.production)' {}
```
The `{}` comes from the `find -exec`. It represents a single file path.

And finally write them to the files they came from:
```
> {}.new.json && mv {}.new.json {}" \;
```
When I tried to `> {}` to redirect the output to the file, it just deleted the contents of the file, so I worked around this by writing to `{}.new.json` and then immediately renaming it to what it should be at `{}`.

### CODE REVIEW

File: src/applications/edu-benefits/10203/manifest.json [hunk 1]
```diff
@@ -2,29 +2,5 @@
   "appName": "22-10203 Education benefits form",
   "entryFile": "./edu-benefits-entry.jsx",
   "entryName": "10203-edu-benefits",
-  "rootUrl": "/education/apply-for-education-benefits/application/10203",
-  "template": {
-    "vagovprod": false,
```
Comment (zurbergram) [inline src/applications/edu-benefits/10203/manifest.json:7]: this form should be disabled in production

Comment (cvalarida) [reply src/applications/edu-benefits/10203/manifest.json:7]: It will continue to be. https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/registry.json#L553

Comment (cvalarida) [reply src/applications/edu-benefits/10203/manifest.json:7]: As I mentioned in Slack, [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/12367) changes the way application landing pages are created. We're using the `registry.json` for the content build to create the landing pages.

**Review by ncksllvn — APPROVED** [review]
Trusting you on this one 😄

**Review by brangi — APPROVED** [review]

**Review by zurbergram — APPROVED** [review]

**Comment by Mottie** [comment]
Aren't the `breadcrumbs_override` values still used? I swear I remember using them recently and they worked.

**Review by Mottie — APPROVED** [review]

**Review by jbalboni — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Comment by cvalarida** [comment]
> Aren't the breadcrumbs_override values still used? I swear I remember using them recently and they worked.

@Mottie The `breadcrumbs_override` are still [included in the templates](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/registry.json#L306-L315) in `registry.json`, which is what's used to create the static HTML landing page for your applications. I'm not 100% sure if they're used right now, but we're still passing them to the Liquid templates like we were before.

**Comment by cvalarida** [comment]
@ncksllvn @brangi @zurbergram @Mottie @jbalboni @micahchiang 

TL;DR: I updated some e2e tests and the corresponding manifest.json files. Please make sure your tests are running in CI if you expect them to, or not running in CI if you don't expect them to.

The longer version:

Looks like some of the e2e tests were using the manifest files to determine whether they should be run in CI. This made a lot of sense, but since we're now using `registry.json` to determine whether the landing page should be in production, AND we're planning on moving the `registry.json` out of `vets-website` when we move the content build out, we won't be able to use the `template[process.env.BUILDTYPE]` in `registry.json` to determine whether the application is in production.

Instead, what I've done is
1. Added `e2eTestsDisabled` to your applications' `manifest.json` files if that `template.vagovprod` was false for your application
2. Updated the tests to `module.exports['@disabled']` if `e2eTestsDisabled` was set to true and you're not trying to run the tests locally

This should still prevent your tests to run in CI if the landing page isn't build for prod.

Moving forward, after the content build is removed from `vets-website`, the plan is to always build a landing page for e2e tests that won't get deployed to production. This way, your applications will always be tested in CI and we won't be limited by whether it's been released in prod yet or not.

**Review by bradl3yC — APPROVED** [review]

File: src/applications/disability-benefits/2346/manifest.json [hunk 2]
```diff
@@ -3,8 +3,5 @@
   "entryFile": "./app-entry.jsx",
   "entryName": "hearing-aid-batteries-and-accessories",
   "rootUrl": "/hearing-aid-batteries-and-accessories",
-  "production": false,
-  "template": {
-    "vagovprod": false
-  }
+  "e2eTestsDisabled": true
```
Comment (rianfowler) [inline src/applications/disability-benefits/2346/manifest.json:6]: long term do we want to nest app specific attributes? I think only build stuff should be root level or maybe nested in its own grouping.

File: src/applications/edu-benefits/tests/0993/00.0093-opt-out.e2e.spec.js [hunk 3]
```diff
@@ -55,5 +54,3 @@ module.exports = E2eHelpers.createE2eTest(client => {
   client.axeCheck('.main');
   client.end();
 });
-
-module.exports['@disabled'] = !manifest.production;
```
Comment (rianfowler) [inline src/applications/edu-benefits/tests/0993/00.0093-opt-out.e2e.spec.js:59]: This is interesting. How would we ensure this isn't being used at runtime?

**Review by rianfowler — APPROVED** [review]

File: src/applications/disability-benefits/2346/manifest.json (hunk 2 above)
Comment (cvalarida) [reply src/applications/disability-benefits/2346/manifest.json:6]: Yeah, I think that'd be a good next iteration. :+1:

File: src/applications/edu-benefits/tests/0993/00.0093-opt-out.e2e.spec.js (hunk 3 above)
Comment (cvalarida) [reply src/applications/edu-benefits/tests/0993/00.0093-opt-out.e2e.spec.js:59]: I'm not sure I understand the question.
